exception_escaping/__init__.py,sha256=kCyYNpgZyV5JjkJ7Kttb6jSdYC7pNEEgtllCaAeVGdM,131
exception_escaping/errors.py,sha256=Eq6POhska5QKjhkBeZZZTYJkyVdEkaOxvck3JOjLrd0,66
exception_escaping/proxy_module.py,sha256=mEZre-D_AUJDjsPU3p1FPar14QyjBPbldIHxCkFCNkA,2167
exception_escaping/py.typed,sha256=47DEQpj8HBSa-_TImW-5JCeuQeRkm5NMpJWZG3hSuFU,0
exception_escaping/wrapper.py,sha256=FiMJrGnVwcG9E6NF30l9fOU9A44gyN9_KR5zT1o0Y00,1762
exception_escaping-0.0.4.dist-info/LICENSE,sha256=ORr8GcdTjVeQVDg9_JqOOPXr0lcv1aEzwu8Qwtc4-g8,1067
exception_escaping-0.0.4.dist-info/METADATA,sha256=IrSFeKihmKqN2QPEK9p7JvqOLKVF6j-tURToY0oxd_4,6483
exception_escaping-0.0.4.dist-info/WHEEL,sha256=oiQVh_5PnQM0E3gPdiz09WCNmwiHDMaGer_elqB3coM,92
exception_escaping-0.0.4.dist-info/top_level.txt,sha256=Yr7kngfzpOzZQY1b_u9WvR5iLDCvTSx42tFfq2G1pWc,19
exception_escaping-0.0.4.dist-info/RECORD,,
